packages/firestore/test/unit/local/lru_garbage_collector.test.ts

```diff
@@ -983,4 +986,50 @@ function genericLruGarbageCollectorTests(
     );
     expect(finalCacheSize).to.be.lessThan(cacheSize);
   });
+
+  it('caps sequence numbers to collect', async () => {
+    //await persistence.shutdown();
```

Comment:
What's this?

Reply:
Removed.

Maybe irrelevant details, but it turns out that with multi-tab, shutting down the persistence layer outside of the async queue can lead to race conditions. The test teardown and initialize methods now do the right thing.